config/jobs/kubernetes/test-infra/test-infra-presubmits.yaml

```diff
@@ -97,3 +97,23 @@ presubmits:
         - --warnings=duplicate-job-refs
     annotations:
       testgrid-dashboards: presubmits-test-infra
+  - name: pull-test-infra-integration
+    branches:
+    - master
+    always_run: false
```

Comment:
If always_run: false and there's no "run when this file is changed" regex, does that mean it'll never run unless you `/test` for it?

Reply:
Yeah that's intended, this job will be used for testing my initial PR of seeding integration tests and before that no other PR should ever trigger this job since they are bound to fail